admin/client.go

```diff
@@ -18,12 +18,12 @@ package admin
 
 import (
 	"context"
+	"github.com/yorkie-team/yorkie/gen/go/yorkie/v1"
```

Comment:
We organized directories according to the [Go Project Layout](https://github.com/golang-standards/project-layout#api).

How about changing this like the below?
```suggestion
	api "github.com/yorkie-team/yorkie/api/v1"
```